multipart.go

```diff
@@ -0,0 +1,176 @@
+package example
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"math/rand"
+
+	"github.com/beyondstorage/go-storage/v4/pairs"
+	"github.com/beyondstorage/go-storage/v4/pkg/randbytes"
+	"github.com/beyondstorage/go-storage/v4/types"
+)
+
+func MultipartUploadTest(store types.Storager, path string) {
```

Comment:
We are not writing tests case, we are writing examples.

Our name should be

- `Multipart`: a full example of multipart API
- `ResumeMultipart`: an example for resume a multipart upload
- `CancelMultipart`: delete a multipart object

In examples

- We don't need to care too much about error handling (`log.Fatalf` is enough)
- We don't need to care about user input (just treat them as valid)
- We don't need to care about garbage collect (user who run examples will handle)

Reply:
Got it.